features/step_definitions/email_steps.rb

```diff
@@ -34,6 +34,17 @@ def current_email_address
     # Note that last_email_address will be reset after each Scenario.
     last_email_address || "affiliate_admin@fixtures.org"
   end
+
+  def searchgov_addresses(address)
+    address = case address
+              when "support"
+                "#{Rails.application.secrets.organization[:support_email_address]}"
+              when "admin"
+                Rails.application.secrets.organization[:admin_email_address]
+              else
+                address
+              end
+  end
```

Comment:
I think we should remove this helper and stick with the hard-coded expectation in the features. This strikes me as inserting the sausage-making process into the test, whereas our integration tests should stick as closely as possible to testing the behavior and not the implementation. For example, you wouldn't write the raw expectation as: `"And #{Rails.application.secrets.organization[:admin_email_address]} should receive an email"`. We're also not referencing those common emails very much in the features, so this wouldn't save us much work if they change in the future.